src/services/ui/src/features/package/package-details/appk.tsx

```diff
@@ -31,10 +33,12 @@ export const AppK = () => {
           setTimeout(() => {
             setRemoveChild("");
             cache.refetch();
+            setLoading(false);
           }, 5000);
         },
         onError: (err) => {
           console.error(err);
+          setLoading(false);
```

Comment:
`onSuccess` and `onError` are part of the available `options` given to `useSubmissionService` and don't need to be set in here as far as I can tell -- we should define this in our hook definition on line `18`